src/locale/locales/fr/messages.po

```diff
@@ -2886,7 +2875,7 @@ msgstr "Stockage ancien effacé, vous devez redémarrer l’application maintena
 
 #: src/components/StarterPack/ProfileStarterPacks.tsx:293
 msgid "Let me choose"
-msgstr ""
+msgstr "Permettez-moi de choisir"
```

Comment:
Let's be more direct (`Permettez-moi` is quite posh in this context I would say :D)

```suggestion
msgstr "Laissez-moi choisir"
```